### DISCUSSION TITLE
Running some cells on Duet basics notebook have to be done twice?

### DISCUSSION THREAD

**Discussion by butchland** [OpenMined/courses#313] [Course: Foundations of Private Computation]
Hi all, 
I was wondering if anyone else is experiencing this weird behavior in some jupyter notebook cells where you have to run them twice in order for the cell to get executed. 

The first time you run it (Ctrl-Enter/Shift-Enter/click on the Run button) the asterisk indicator is on for a few seconds (10-30 secs)  then it clears it (just like it was never executed at all). 

The second time is when the cell is run, its usually fast and that's when the cell has its execution count  updated.

This happens for both the Duet_Basics_Data_Owner and the Duet_Basics_Data_Scientists notebook.

This is on the Lesson 1, Concept 14 Duet Basics of the Federated Learning module.

I just followed the instructions to create a new conda env as per instructions on the pysyft github page and loaded the requirements.txt from the git repo for the course. Also installed the latest pysyft from the github repo as per instructions in the course.

I'm running on MacOS Mojave, on a Brave browser, just in case that matters.

**Comment by znreza** [comment]
Hi @butchland while preparing the contents I myself have faced this issue once as well in MacOS. I am really not sure what caused this issue but it looked like a Jupyter notebook with MacOS issue to me. I would suggest restarting and reconnecting your notebooks as I never faced this issue again after doing so.
